tests/CMakeLists.txt

```
@@ -102,8 +102,12 @@ endif ()
 
 ## -- Benchmark
 
-add_executable(broker-benchmark benchmark/broker-benchmark.cc)
-target_link_libraries(broker-benchmark ${libbroker})
-
-add_executable(broker-cluster-benchmark benchmark/broker-cluster-benchmark.cc)
-target_link_libraries(broker-cluster-benchmark ${libbroker})
+if (BROKER_ENABLE_BENCHMARKS)
+  add_executable(broker-benchmark benchmark/broker-benchmark.cc)
+  target_link_libraries(broker-benchmark ${libbroker})
+  install(FILES ${CMAKE_BINARY_DIR}/bin/broker-benchmark DESTINATION bin)
```

```suggestion
  install(TARGETS broker-benchmark DESTINATION bin)
```

Don't immediately remember if there's substantial difference, but at least a bit shorter this way.